cfme/tests/cloud_infra_common/test_vm_ownership.py

```diff
@@ -0,0 +1,166 @@
+import cfme.configure.access_control as ac
+import fauxfactory
+import pytest
+from cfme import Credential, login
+from cfme.common.vm import VM
+from utils.providers import setup_a_provider
+from utils import testgen
+
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = testgen.all_providers(
+        metafunc, 'ownership_vm', require_fields=True)
+    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist, scope="module")
+
+
+@pytest.fixture(scope="module")
```

Comment:
This shouldn't be required, as the "setup_provider" should be used alongside testgen.